script.js

```diff
@@ -47,13 +83,19 @@ function drawNewGrid() {
         drawNewGrid();
     }
 
-    else {
-        return buildGrid(newSize);
-    }
+    return buildGrid(newSize);
 }
 
+/**
+ * a.) can you think of why this might be an unsafe operation?
+ * b.) it would be cool to also reset your background colors on your tiles when you did this
+ */
```

Comment:
In the future, I plan to implement a color picker for the user! In an old commit, I did have it written to wipe the background color, but I thought it unnecessary since I was tearing the grid down to start over with a new appended DOM.